### DISCUSSION TITLE
Menu Bar does not disappear reliably

### DISCUSSION THREAD

**Discussion by mer-at** [AppGround-io/bluetooth-keyboard-and-mouse-support#137] [Bugs & Problems]
**Describe the bug**
when using the trackpad only, sometimes the menu bar is visible and does not reliably disappear.

**To Reproduce**
1. open app
2. go to the mouse/trackpad
3a. if the menu bar is not visible, just rotate the device once.
4a. sometimes the menu bar stays visible after that, sometimes it doesnt

3b. "pull down" from the top of the screen to display the menu bar
4b. no way of getting rid of the menu bar anymore
5b. rotate the device multiple times between landscape and portrait mode, and eventually it will be hidden again

**Expected behavior**
the menu bar disappears after X seconds of not using it
OR a way to hide the menu bar manually

**Comment by mer-at** [comment]
Here's a video of that strange behaviour: https://www.youtube.com/watch?v=QM9gqPJm2yA&feature=youtu.be

**Comment by zorazor** [comment]
I think this is caused by the fullscreen.
The fullscreen-mode is unfortunately slightly buggy, e.g. also in use with the keyboard.

**Comment by AppGround-io** [comment]
I'm working on improvements in this area, but there are a lot of edge cases and issues with different versions of Android. This should be fixed soon though, I'm currently waiting for some improvements to be made to system libraries.
